src/ext/backtrace.c

```diff
@@ -1,11 +1,12 @@
-#if defined(__GLIBC__) || defined(__APPLE__)
 #include <execinfo.h>
```

Comment:
Can we move this include inside of the ifdef? :)

Reply:
done